src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -0,0 +1,1914 @@
+/*
```

Comment:
Still need to remove these licenses (but can do this in a separate PR).

Reply:
Fixed!